babel.config.js

```diff
@@ -1,6 +1,5 @@
 /* eslint-disable import/no-commonjs */
 module.exports = {
-  presets: [['@babel/env', require('./packages/@sanity/server/src/configs/babel-env-config')]],
-  plugins: ['lodash', '@babel/plugin-proposal-class-properties'],
+  ...require('./.babelrc'),
```

Comment:
This needs to use `Object.assign` to be compatible with node 6

Reply:
Ah, good point, thanks!